client/index.html

```diff
@@ -218,38 +218,20 @@ <h1 class="title">Settings</h1>
 							</div>
 							<div class="col-sm-6">
 								<label class="opt">
-									<input type="checkbox" name="join">
-									Show joins
+									<input type="checkbox" name="optional">
+									Show <abbr title="Joins, parts, kicks, nick changes, and mode changes">status messages</abbr>
```

Comment:
Actually, I did something so that I wouldn't have to repeat that question mark icon, or just put it once but being awkward. Same screenshot as comment above, but putting it here in case GitHub messes up comment order:

<img width="468" alt="screen shot 2017-08-06 at 00 49 39" src="https://user-images.githubusercontent.com/113730/29000957-c71fef54-7a49-11e7-836c-381b30c04b4e.png">